2025-05-16 10:02:59 - app2_ia.utils.limpieza - ERROR - Modelo 'es_core_news_sm' no encontrado. Por favor, descárgalo ejecutando: python -m spacy download es_core_news_sm. El procesamiento de texto no funcionará sin el modelo.
2025-05-16 10:03:21 - fastapi - ERROR - Form data requires "python-multipart" to be installed. 
You can install "python-multipart" with: 

pip install python-multipart

2025-05-16 10:03:51 - app2_ia.utils.limpieza - ERROR - Modelo 'es_core_news_sm' no encontrado. Por favor, descárgalo ejecutando: python -m spacy download es_core_news_sm. El procesamiento de texto no funcionará sin el modelo.
2025-05-16 10:05:38 - app2_ia.utils.limpieza - ERROR - Modelo 'es_core_news_sm' no encontrado. Por favor, descárgalo ejecutando: python -m spacy download es_core_news_sm. El procesamiento de texto no funcionará sin el modelo.
2025-05-16 10:07:01 - app2_ia.services.ingest_service - INFO - Log de errores guardado en logs\errores_20250516_100701.log
2025-05-16 10:13:33 - app2_ia.services.ingest_service - ERROR - Error al parsear fila {'candidato_id': 1001, 'puesto': 'Marketing', 'fortalezas': 'Creatividad', 'debilidades': 'Impulsividad', 'valoracion_gpt': 'El candidato tiene buenas ideas, aunque debe mejorar su enfoque.'}: 1 validation error for CandidatoCrudo
candidato_id
  Input should be a valid string [type=string_type, input_value=1001, input_type=int]
    For further information visit https://errors.pydantic.dev/2.11/v/string_type
2025-05-16 10:13:33 - app2_ia.services.ingest_service - ERROR - Error al parsear fila {'candidato_id': 1002, 'puesto': 'Ventas', 'fortalezas': 'Estrategia', 'debilidades': 'Desorganización', 'valoracion_gpt': 'Muy buen trato con clientes, debe mejorar la planificación.'}: 1 validation error for CandidatoCrudo
candidato_id
  Input should be a valid string [type=string_type, input_value=1002, input_type=int]
    For further information visit https://errors.pydantic.dev/2.11/v/string_type
2025-05-16 10:13:33 - app2_ia.services.ingest_service - ERROR - Error al parsear fila {'candidato_id': 1003, 'puesto': 'RRHH', 'fortalezas': 'Empatía', 'debilidades': 'Inseguridad', 'valoracion_gpt': 'Gran capacidad de escucha, aunque a veces duda al hablar en público.'}: 1 validation error for CandidatoCrudo
candidato_id
  Input should be a valid string [type=string_type, input_value=1003, input_type=int]
    For further information visit https://errors.pydantic.dev/2.11/v/string_type
2025-05-16 10:13:33 - app2_ia.services.ingest_service - INFO - Log de errores guardado en logs\errores_20250516_101333.log
2025-05-16 10:15:38 - app2_ia.utils.limpieza - ERROR - Modelo 'es_core_news_sm' no encontrado. Por favor, descárgalo ejecutando: python -m spacy download es_core_news_sm. El procesamiento de texto no funcionará sin el modelo.
2025-05-16 10:15:50 - app2_ia.utils.limpieza - WARNING - Intento de procesar texto ('El candidato tiene b...') pero el modelo de spaCy no está cargado. La función devolverá un string vacío.
2025-05-16 10:15:50 - sentence_transformers.SentenceTransformer - INFO - Use pytorch device_name: cpu
2025-05-16 10:15:50 - sentence_transformers.SentenceTransformer - INFO - Load pretrained SentenceTransformer: all-mpnet-base-v2
2025-05-16 10:17:06 - app2_ia.services.embedding - INFO - Modelo base cargado (768D) con proyección a 1536D
2025-05-16 10:17:06 - app2_ia.services.embedding - INFO - Módulo 3 inicializado - Embeddings de 1536 dimensiones
2025-05-16 10:17:06 - app2_ia.services.ingest_service - INFO - Candidato 1001 insertado en VectorDB
2025-05-16 10:17:06 - app2_ia.utils.limpieza - WARNING - Intento de procesar texto ('Muy buen trato con c...') pero el modelo de spaCy no está cargado. La función devolverá un string vacío.
2025-05-16 10:17:07 - app2_ia.services.ingest_service - INFO - Candidato 1002 insertado en VectorDB
2025-05-16 10:17:07 - app2_ia.utils.limpieza - WARNING - Intento de procesar texto ('Gran capacidad de es...') pero el modelo de spaCy no está cargado. La función devolverá un string vacío.
2025-05-16 10:17:07 - app2_ia.services.ingest_service - INFO - Candidato 1003 insertado en VectorDB
2025-05-16 10:17:57 - app2_ia.services.ingest_service - INFO - Log de errores guardado en logs\errores_20250516_101757.log
2025-05-16 11:52:45 - app2_ia.services.ingest_service - INFO - Log de errores guardado en logs\errores_20250516_115245.log
2025-05-16 11:52:47 - app2_ia.services.ingest_service - INFO - Log de errores guardado en logs\errores_20250516_115247.log
2025-05-16 11:56:24 - app2_ia.utils.limpieza - ERROR - Modelo 'es_core_news_sm' no encontrado. Por favor, descárgalo ejecutando: python -m spacy download es_core_news_sm. El procesamiento de texto no funcionará sin el modelo.
2025-05-16 11:57:08 - app2_ia.utils.validacion - WARNING - Fila 2: dato sensible detectado
2025-05-16 11:57:08 - app2_ia.utils.validacion - WARNING - Fila 3: campos ['candidato_id', 'puesto', 'fortalezas', 'debilidades'] no pueden estar vacíos
2025-05-16 11:57:08 - app2_ia.utils.validacion - WARNING - Fila 4: ID no numérico ('100X')
2025-05-16 11:57:08 - app2_ia.services.ingest_service - INFO - Log de errores guardado en logs\errores_20250516_115708.log
2025-05-16 11:57:08 - app2_ia.utils.limpieza - WARNING - Intento de procesar texto ('Muy buen trato con c...') pero el modelo de spaCy no está cargado. La función devolverá un string vacío.
2025-05-16 11:57:08 - sentence_transformers.SentenceTransformer - INFO - Use pytorch device_name: cpu
2025-05-16 11:57:08 - sentence_transformers.SentenceTransformer - INFO - Load pretrained SentenceTransformer: all-mpnet-base-v2
2025-05-16 11:57:14 - app2_ia.services.embedding - INFO - Modelo base cargado (768D) con proyección a 1536D
2025-05-16 11:57:14 - app2_ia.services.embedding - INFO - Módulo 3 inicializado - Embeddings de 1536 dimensiones
2025-05-16 11:57:15 - app2_ia.services.ingest_service - INFO - Candidato 1003 insertado en VectorDB
2025-05-16 11:57:15 - app2_ia.utils.limpieza - WARNING - Intento de procesar texto ('Gran capacidad de es...') pero el modelo de spaCy no está cargado. La función devolverá un string vacío.
2025-05-16 11:57:15 - app2_ia.services.ingest_service - INFO - Candidato 1004 insertado en VectorDB
2025-05-16 11:57:15 - app2_ia.utils.limpieza - WARNING - Intento de procesar texto ('Estrategia Desorgani...') pero el modelo de spaCy no está cargado. La función devolverá un string vacío.
2025-05-16 11:57:16 - app2_ia.services.ingest_service - INFO - Candidato 1005 insertado en VectorDB
2025-05-16 11:57:16 - app2_ia.utils.limpieza - WARNING - Intento de procesar texto ('Gran capacidad de es...') pero el modelo de spaCy no está cargado. La función devolverá un string vacío.
2025-05-16 11:57:16 - app2_ia.services.ingest_service - INFO - Candidato 1006 insertado en VectorDB
2025-05-16 11:57:16 - app2_ia.utils.limpieza - WARNING - Intento de procesar texto ('Muy buen trato con c...') pero el modelo de spaCy no está cargado. La función devolverá un string vacío.
2025-05-16 11:57:17 - app2_ia.services.ingest_service - INFO - Candidato 1007 insertado en VectorDB
2025-05-16 12:04:09 - app2_ia.utils.limpieza - ERROR - Modelo 'es_core_news_sm' no encontrado. Por favor, descárgalo ejecutando: python -m spacy download es_core_news_sm. El procesamiento de texto no funcionará sin el modelo.
2025-05-16 12:04:17 - app2_ia.utils.limpieza - ERROR - Modelo 'es_core_news_sm' no encontrado. Por favor, descárgalo ejecutando: python -m spacy download es_core_news_sm. El procesamiento de texto no funcionará sin el modelo.
2025-05-16 12:04:36 - app2_ia.utils.limpieza - ERROR - Modelo 'es_core_news_sm' no encontrado. Por favor, descárgalo ejecutando: python -m spacy download es_core_news_sm. El procesamiento de texto no funcionará sin el modelo.
2025-05-16 12:05:46 - app2_ia.utils.validacion - WARNING - Fila 2: dato sensible detectado
2025-05-16 12:05:46 - app2_ia.utils.validacion - WARNING - Fila 3: campos ['candidato_id', 'puesto', 'fortalezas', 'debilidades'] no pueden estar vacíos
2025-05-16 12:05:46 - app2_ia.utils.validacion - WARNING - Fila 4: ID no numérico ('100X')
2025-05-16 12:05:46 - app2_ia.services.ingest_service - INFO - Log de errores guardado en logs\errores_20250516_120546.log
2025-05-16 12:05:46 - app2_ia.utils.limpieza - WARNING - Intento de procesar texto ('Muy buen trato con c...') pero el modelo de spaCy no está cargado. La función devolverá un string vacío.
2025-05-16 12:05:46 - sentence_transformers.SentenceTransformer - INFO - Use pytorch device_name: cpu
2025-05-16 12:05:46 - sentence_transformers.SentenceTransformer - INFO - Load pretrained SentenceTransformer: all-mpnet-base-v2
2025-05-16 12:05:49 - app2_ia.services.embedding - INFO - Modelo base cargado (768D) con proyección a 1536D
2025-05-16 12:05:49 - app2_ia.services.embedding - INFO - Módulo 3 inicializado - Embeddings de 1536 dimensiones
2025-05-16 12:05:50 - app2_ia.services.ingest_service - INFO - Candidato 1003 insertado en VectorDB
2025-05-16 12:05:50 - app2_ia.utils.limpieza - WARNING - Intento de procesar texto ('Gran capacidad de es...') pero el modelo de spaCy no está cargado. La función devolverá un string vacío.
2025-05-16 12:05:50 - app2_ia.services.ingest_service - INFO - Candidato 1004 insertado en VectorDB
2025-05-16 12:05:50 - app2_ia.utils.limpieza - WARNING - Intento de procesar texto ('Estrategia Desorgani...') pero el modelo de spaCy no está cargado. La función devolverá un string vacío.
2025-05-16 12:05:51 - app2_ia.services.ingest_service - INFO - Candidato 1005 insertado en VectorDB
2025-05-16 12:05:51 - app2_ia.utils.limpieza - WARNING - Intento de procesar texto ('Gran capacidad de es...') pero el modelo de spaCy no está cargado. La función devolverá un string vacío.
2025-05-16 12:05:51 - app2_ia.services.ingest_service - INFO - Candidato 1006 insertado en VectorDB
2025-05-16 12:05:51 - app2_ia.utils.limpieza - WARNING - Intento de procesar texto ('Muy buen trato con c...') pero el modelo de spaCy no está cargado. La función devolverá un string vacío.
2025-05-16 12:05:52 - app2_ia.services.ingest_service - INFO - Candidato 1007 insertado en VectorDB
